muser/src/main/java/edu/usf/cutr/mfe/ProcessorMain.java

```diff
@@ -28,7 +28,7 @@ public static void main(String[] args) {
            System.err.println("Invalid command line options");
        }
 		//programOptions.setKeyFilePath();
-	    //programOptions.setUserId(null);
+	   // programOptions.setUserId(null);
```

Comment:
You could go ahead and remove any commented code before you push. Since you have committed the file, you can always grab the code back from previous commits. Better yet, if you're using one of JetBrains' IDEs, you can right-click on the file and go to Local History for any recent changes.